src/Umbraco.Infrastructure/Services/Implement/CacheInstructionService.cs

```diff
@@ -247,6 +247,13 @@ private int ProcessDatabaseInstructions(CacheRefresherCollection cacheRefreshers
         /// </summary>
         private bool TryDeserializeInstructions(CacheInstruction instruction, out JArray jsonInstructions)
```

Comment:
While debugging the original issue, I also came across a 
```
System.ArgumentNullException:
   at Newtonsoft.Json.JsonConvert.DeserializeObject (Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed)
   at Newtonsoft.Json.JsonConvert.DeserializeObject (Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed)
   at Newtonsoft.Json.JsonConvert.DeserializeObject (Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed)
   at Umbraco.Cms.Core.Services.Implement.CacheInstructionService.TryDeserializeInstructions (Umbraco.Infrastructure, Version=9.3.1.0, Culture=neutral, PublicKeyToken=null)
   at Umbraco.Cms.Core.Services.Implement.CacheInstructionService.ProcessDatabaseInstructions (Umbraco.Infrastructure, Version=9.3.1.0, Culture=neutral, PublicKeyToken=null)
   at Umbraco.Cms.Core.Services.Implement.CacheInstructionService.ProcessInstructions (Umbraco.Infrastructure, Version=9.3.1.0, Culture=neutral, PublicKeyToken=null)
   at Umbraco.Cms.Infrastructure.Sync.DatabaseServerMessenger.Sync (Umbraco.Infrastructure, Version=9.3.1.0, Culture=neutral, PublicKeyToken=null)
   at Umbraco.Cms.Infrastructure.HostedServices.ServerRegistration.InstructionProcessTask.PerformExecuteAsync (Umbraco.Infrastructure, Version=9.3.1.0, Culture=neutral, PublicKeyToken=null)
```
While I'm not sure how a `CacheInstruction` with null instructions can be generated, I've addressed it so the method behavior matches the name and what would be expected.